### PR TITLE
change image to alpine

### PR DESCRIPTION
**Pull request by gianrubio**
Fix kube-lego crashing #71 

Same as https://github.com/funktionio/funktion/pull/32

The image size just increased 4MB
```
kube-lego-alpine                                  latest              ea7ca4b996df        11 hours ago        65 MB
kube-lego-scratch                                 latest              36e6953dc0f6        12 hours ago        61 MB
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM scratch
+FROM alpine
```
Comment (simonswine) [inline Dockerfile]: Can you use a tagged version here?
`alpine:3.5`

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 ADD dist/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```
Comment (simonswine) [inline Dockerfile]: This certs should be coming from alpine then:

RUN apk --update add ca-certificates && rm -rf /var/cache/apk/*

**Comment by gianrubio** [comment]
@simonswine I did, could you review again?

**Comment by simonswine** [comment]
LGTM! Thanks

**Pull request merged**